packages/tenderly-actions/tenderly.yaml

```diff
@@ -0,0 +1,52 @@
+account_id: "VenusTools"
+actions:
+  VenusTools/tenderly-actions:
+    runtime: v2
+    sources: src/actions
+    specs:
+      processRiskParameterUpdate:
+        description: Action to Process RiskOrcale ParameterUpdate to RiskSteward
+        function: processRiskParameterUpdate:processRiskParameterUpdateFn
+        execution_type: sequential
+        trigger:
+          type: transaction
+          transaction:
+            status:
+              - mined
+            filters:
+              - network: 97 # BSC Testnet
+                eventEmitted:
+                  contract:
+                    address: 0x7BD97DD6C199532d11Cf5f55E13a120dB6dd0F4F
+                  name: ParameterUpdated
+              - network: 421614 # Arbitrum Sepolia
+                eventEmitted:
+                  contract:
+                    address: 0xE6b7B1846106605fdfaB3a9F407dd64bed6917a6
+                  name: ParameterUpdated
+              - network: 84532 # Base Sepolia
+                eventEmitted:
+                  contract:
+                    address: 0xf69fd7757c8A59DFA5c35622d9D44B31dB21B0a2
+                  name: ParameterUpdated
+              - network: 11155420 # OP Sepolia
+                eventEmitted:
+                  contract:
+                    address: 0x4fCbfE445396f31005b3Fd2F6DE2A986d6E2dCB5
+                  name: ParameterUpdated
+              - network: 11155111 # Sepolia
+                eventEmitted:
+                  contract:
+                    address: 0xDF1807d323Dd2E2871e8b394c8F7595E097e1C73
+                  name: ParameterUpdated
+              - network: 1301 # Unichain Sepolia
+                eventEmitted:
+                  contract:
+                    address: 0x4fCbfE445396f31005b3Fd2F6DE2A986d6E2dCB5
+                  name: ParameterUpdated
+              - network: 300 # ZKSync Sepolia
```

Comment:
Would it make sense to add now the available config for Ethereum? Or do you prefer to wait?